data/ga/legislature/Sam-Park-bcab0f3a-68fd-4f91-bbc0-bb68b122c044.yml

```diff
@@ -0,0 +1,38 @@
+id: ocd-person/bcab0f3a-68fd-4f91-bbc0-bb68b122c044
+name: Sam Park
+given_name: Sam
+family_name: Park
+email: sam.park@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/park-sam-4901.jpg
+party:
+- name: Democratic
+roles:
```

Comment:
@sroomf this person served in the house district 101 from January 2017 until  01/09/2023.